docs/TestingAsyncCode.md

```diff
@@ -48,15 +48,13 @@ If `done()` is never called, the test will fail, which is what you want to happe
 
 ### Promises
```

Comment:
Hey @cpojer. Could you elaborate a bit more what you want written here? In the `TutorialAsync.md` file we removed a section that contained `expect.assertions()`, would you like to add this back into the documentation so that people who use older versions of jest can still reference the docs for async problems?